src/utils/reclient.js

```diff
@@ -99,11 +99,22 @@ function reclientEnv(config) {
     return {};
   }
 
-  return {
+  let reclientEnv = {
     RBE_service: config.reclientServiceAddress || rbeServiceAddress,
     RBE_experimental_credentials_helper: getHelperPath(config),
     RBE_experimental_credentials_helper_args: 'print',
   };
+
+  const result = childProcess.spawnSync(reclientHelperPath, ['status'], {
+    stdio: 'pipe',
+  });
+  if (result.status === 0) {
+    const regexMatch = result.stdout.toString().match(/^Access:\s*(.*)$/gm);
```

Comment:
We own the helper, we can do better than a regex surely 😂 maybe a new command `e d rbe flags` which prints JSON encoded flags for us to set as env vars?

Reply:
sigh...